src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1708,10 +1726,11 @@ static void lb_on_server_status_received_locked(grpc_exec_ctx* exec_ctx,
     char* status_details =
         grpc_slice_to_c_string(glb_policy->lb_call_status_details);
     gpr_log(GPR_INFO,
-            "Status from LB server received. Status = %d, Details = '%s', "
+            "[grpclb %p] Status from LB server received. Status = %d, Details "
+            "= '%s', "
             "(call: %p), error %p",
```

Comment:
Logging the address of the error doesn't seem very useful.  Why not log `grpc_error_string(error)` instead?

Reply:
Done